src/features/FileIOAdapter.js

```diff
@@ -142,22 +149,16 @@ export class FileIOAdapter {
   }
 
   writeAnswers = async (data) => {
-    console.log("participantId: " + data.participantId);
-    console.log("csv: " + data.csv);
     let postSurveyAnswersStr = JSON.stringify(data.postSurveyAnswers, null, 2);
-    console.log("postSurveyAnswers: " + postSurveyAnswersStr);
 
     const fileNameAnswers = "answers-" + data.participantId + ".csv";
     const fileNamePostSurveyAnswers =
       "post-survey-answers-" + data.participantId + ".json";
-    console.log("fileNameAnswers: " + fileNameAnswers);
 
     if (process.env.REACT_APP_AWS_ENABLED) {
-      console.log("AWS ENABLED");
       uploadFile(fileNameAnswers, data.csv);
       uploadFile(fileNamePostSurveyAnswers, postSurveyAnswersStr);
     } else {
-      console.log("AWS DISABLED");
```

Comment:
forgot about these `console.log` statements but these 3 might be nice to keep in so that we know its working in prod